File: RuntimeData/xnaruntime/RuntimeXNA/RuntimeXNA/Events/CQualToOiList.cs
//----------------------------------------------------------------------------------
//
// CQUALTOOI : qualifiers
//
//----------------------------------------------------------------------------------
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
namespace RuntimeXNA.Events
{
	
	public class CQualToOiList
	{		
		public short qoiCurrentOi = 0;
		public short qoiNext = 0;
		public short qoiActionPos = 0;
		public int qoiCurrentRoutine = 0;
		public int qoiActionCount = 0;
		public int qoiActionLoopCount = 0;
		public bool qoiNextFlag = false;
		public bool qoiSelectedFlag = false;
		public short[] qoiList = null; // Array OINUM / OFFSET
		
		public CQualToOiList()
		{
		}
	}
}

File: RuntimeData/runtime/app/src/main/java/Events/CQualToOiList.java
/* Copyright (c) 1996-2013 Clickteam
 *
 * This source code is part of the Android exporter for Clickteam Multimedia Fusion 2.
 * 
 * Permission is hereby granted to any person obtaining a legal copy 
 * of Clickteam Multimedia Fusion 2 to use or modify this source code for 
 * debugging, optimizing, or customizing applications created with 
 * Clickteam Multimedia Fusion 2.  Any other use of this source code is prohibited.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 * IN THE SOFTWARE.
 */
//----------------------------------------------------------------------------------
//
// CQUALTOOI : qualifiers
//
//----------------------------------------------------------------------------------
package Events;

public class CQualToOiList 
{

    public short qoiCurrentOi=0;
    public short qoiNext=0;
    public short qoiActionPos=0;
    public int qoiCurrentRoutine=0;
    public int qoiActionCount=0;
    public int qoiActionLoopCount=0;
    public boolean qoiNextFlag=false;
    public boolean qoiSelectedFlag=false;
    public short qoiList[]=null;						// Array OINUM / OFFSET

    public CQualToOiList() 
    {
    }
    
}
